tests/bot/exts/moderation/test_silence.py

```diff
@@ -433,10 +433,10 @@ async def test_sent_correct_message(self):
 
         targets = (MockTextChannel(), MockVoiceChannel(), None)
 
-        for (duration, message, was_silenced), target in itertools.product(test_cases, targets):
+        for i, ((duration, message, was_silenced), target) in enumerate(itertools.product(test_cases, targets)):
             with (
                 mock.patch.object(self.cog, "_set_silence_overwrites", return_value=was_silenced),
-                self.subTest(was_silenced=was_silenced, target=target, message=message),
+                self.subTest(test_case=i, was_silenced=was_silenced, has_target=target is not None),
                 mock.patch.object(self.cog, "send_message") as send_message
             ):
```

Comment:
Adding a test case ID seems silly here, as it's coming from an `itertools.product` and it would be a pain to work out the inputs that it's referring to.

Also the `has_targets` is not a replacement for the `target` because there are multiple not none targets.

I'd accept that the message can be removed though, since it's an expected output.